<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PltrDialogClass</class>
 <widget class="QDialog" name="PltrDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Plant Treatment</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/res.qrc">
    <normaloff>:/pic/pltr16.png</normaloff>:/pic/pltr16.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Name:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Unit:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>kg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>g</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>l</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ml</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>St.</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Active Ingredients:</string>
        </property>
        <property name="buddy">
         <cstring>listViewWirkstoff</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QListView" name="listViewWirkstoff">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Types:</string>
        </property>
        <property name="buddy">
         <cstring>listViewTyp</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QListView" name="listViewTyp">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PltrDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PltrDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
